Corzbank.Web/ClientApp/src/app/data/services/storage.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@angular/core';
+import { CookieService } from 'ngx-cookie-service';
+import { StorageTypeEnum } from '../enums/storage-type.enum';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class StorageService{
+
+    constructor(private cookieService:CookieService) {
+    }
+
+    setItem(storageType:StorageTypeEnum, key:string, value:string){
+        if(storageType==1){
```

Comment:
1. Do not hardcode. StorageTypeEnum.Cookies (instead of 0)
2. Do not use 2 equals sign where it is not necessary. ===
3.  Formatting. Spaces.